storage/engine/data.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2014 The Cockroach Authors.
```

Comment:
on second thought, i think you can put this in mvcc.go and rename the type to `MVCCKey` as suggested by @spencerkimball